01_fib/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(num int) {
+	num1 := 0
+	num2 := num1 + 1
```

Comment:
You can fold this into one line: `num1, num2 := 0, 1`.

Reply:
Done!